treeherder/perf/tasks.py

```diff
@@ -10,3 +13,61 @@ def generate_alerts(signature_id):
     newrelic.agent.add_custom_parameter("signature_id", str(signature_id))
     signature = PerformanceSignature.objects.get(id=signature_id)
     generate_new_alerts_in_series(signature)
+
+
+@task(name='check-confirming-perf-alerts')
+def check_confirming_perf_alerts_status():
+    # TODO: what's this for? do I need it?
+    # newrelic.agent.add_custom_parameter("signature_id", str(signature_id))
+    one_day = timedelta(hours=24)
```

Comment:
I would define this as something like `PERFHERDER_CONFIRMING_INTERVAL` in the settings.